api/blueprint/services_api.go

```diff
@@ -29,7 +29,7 @@ func NewServicesService(concertoService utils.ConcertoService) (*ServicesService
 func (ss *ServicesService) GetServiceList() (services []types.Service, err error) {
 	log.Debug("GetServiceList")
 
-	data, status, err := ss.concertoService.Get("/v1/blueprint/services")
+	data, status, err := ss.concertoService.Get("/v2/blueprint/services")
```

Comment:
This route is one of the ones that has been discontinued. What you should do is turn the service/service_versions commands into cookbook_versions commands that work with /v2/blueprint/cookbook_versions routes

Reply:
Done